SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-only

For some reason (old version of sed/bash?), this falsepos sed thing uses a
lot of RAM within bash and causes an overflow which is caught by bash
but prompts it to crash. As we are not using falsepos detection anyway it
is unused and hence can be set to an empty value.

--- deblob-check	2021-07-30 09:28:09.324276561 +1000
+++ deblob-check	2021-07-30 09:28:31.742449255 +1000
@@ -7232,9 +7232,7 @@
 # $4 is the action for every complete input pattern.
 
 set_sed_main () {
-  falsepos=`${SED-sed} -n 's,^[+]\^*,,p' < "$regex_name" |
-    ${SED-sed} -n -e 's,[$]$,\\\\([\\\\n]\\\\|$\\\\),' \
-	-e '1h; 1!H; ${g;s,[\n],\\\\|,g;s,^\(..*\)$,\\\\(\1\\\\),;p;}'`
+  falsepos=
   blobs=`${SED-sed} -n 's,^[-],,p' < "$regex_name" |
     ${SED-sed} -n -e 's,[$]$,\\\\([\\\\n]\\\\|$\\\\),' \
 	-e '1h; 1!H; ${g;s,[\n],\\\\|,g;s,^\(..*\)$,\\\\(\1\\\\),;p;}'`
